porch/api/porch/v1alpha1/types_packagerevisionresources.go

```diff
@@ -54,3 +54,9 @@ type PackageRevisionResourcesSpec struct {
 	// Resources are the content of the package.
 	Resources map[string]string `json:"resources,omitempty"`
 }
+
+// PackageRevisionResourcesStatus represents state of the rendered package resources.
+type PackageRevisionResourcesStatus struct {
+	// RenderStatus contains the result of rendering the package resources.
+	RenderStatus RenderStatus `json:"renderStatus,omitempty"`
```

Comment:
So this only contains the result after a push right? We don't actually store the information anywhere, so a subsequent `get` will not contain the error? I would expect that he results from the latest run would be returned on every `get`. Similar to the previous comment, I would expect this to be on the `PackageRevision` rather than the `PackageRevisionResources` type.

Reply:
subsequent `get` will not contain the error. I started with this API because this is the API that get's called when user edits a package and that's the place renderStatus is most useful. And I agree that `renderStatus` should be added to `packagerevision`'s status and should also be persisted so that we get it on every `get`. And that can be done as a follow up incrementally.